DTAdmin/Models/Structures/ResultStructure.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ResultStructure.swift
+//  DTAdmin
+//
+//  Created by Yurii Krupa on 11/7/17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import Foundation
+
+struct ResultStructure: Serializable {
```

Comment:
Please rename `ResultStructure` to something more meaningful and not use Structure suffix